Library/.rubocop.yml

```diff
@@ -6,6 +6,8 @@ plugins:
       plugin_class_name: RuboCop::Performance::Plugin
   - rubocop-rspec:
       plugin_class_name: RuboCop::RSpec::Plugin
+  - rubocop-sorbet:
+      plugin_class_name: RuboCop::Sorbet::Plugin
 
 require:
```

Comment:
The old `rubocop-sorbet` line should be removed from the `require` block

Reply:
Done, thanks.